docs/operate/customize/dois.md

```diff
@@ -130,24 +136,20 @@ INVENIO_RDM_PERSISTENT_IDENTIFIERS = {
     },
 }
 ```
-You [can view the default configuration in invenio-rdm-records](https://github.com/inveniosoftware/invenio-rdm-records/blob/e64dd0b81757a391584e63d162d5e6caf6780637/invenio_rdm_records/config.py#L322)
+You [can view the default configuration in invenio-rdm-records](https://github.com/inveniosoftware/invenio-rdm-records/blob/7869cb3242a7feb311f97b633af095edeff4de76/invenio_rdm_records/config.py#L393C1-L410C2)
```

Comment:
So that we don't have to update the link over time.

```suggestion
You [can view the default configuration for `RDM_PERSISTENT_IDENTIFIERS` in invenio-rdm-records](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/config.py)
```